pay-api/tests/unit/api/base_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A helper test.
+
+Test-Suite to ensure that the /payments endpoint is working as expected.
+"""
+
+from pay_api.utils.enums import Role
+
+
+token_header = {
+    'alg': 'RS256',
+    'typ': 'JWT',
+    'kid': 'sbc-auth-cron-job'
+}
+
+
+def get_claims(role: str = Role.BASIC.value):
+    """Return the claim with the role param."""
+    claim = {
+        'jti': 'a50fafa4-c4d6-4a9b-9e51-1e5e0d102878',
+        'exp': 31531718745,
+        'iat': 1531718745,
+        'iss': 'https://sso-dev.pathfinder.gov.bc.ca/auth/realms/fcf0kpqr',
```

Comment:
Will this always come from the dev instance? Should this be pulled from env?

Reply:
I think so..But as an improvement , can be configured in the environment variables/configs as well...
@pksumesh thoughts?